ccloud-observability/docs/general-scenarios/request-rate-limits.rst

```diff
@@ -0,0 +1,32 @@
+.. _ccloud-observability-general-request-rate-limits:
+
+Request rate limits
+********************
+
+If request rate limits are hit, requests may be refused and clients may be throttled to keep the cluster stable. This throttling
+would register as non-zero values for the producer client ``produce-throttle-time-max`` and
+``produce-throttle-time-avg`` metrics and consumer client ``fetch-throttle-time-max`` and ``fetch-throttle-time-avg`` metrics.
+
+#. Open `Grafana <localhost:3000>`__ and use the username ``admin`` and password ``password`` to login.
+
+#. Navigate to the ``Confluent Cloud`` dashboard.
+
+#. Check the ``Requests (rate)`` panel. If this panel is yellow, you have used 80% of your allowed requests; if it's red, you have used 90%.
+
+   |Confluent Cloud Panel|
+
+#. Scroll lower down on the dashboard to see a breakdown of where the requests are to in the ``Request rate`` stacked column chart.
+
+   |Confluent Cloud Request Breakdown|
+
+#. Reduce requests by adjusting producer batching configurations (``linger.ms``), consumer
+   batching configurations (``fetch.max.wait.ms``), and shut down inactive clients.
```

Comment:
If clients are inactive, they shouldn't be requesting anything?

Reply:
Addressed in https://github.com/confluentinc/examples/pull/952/commits/67d43da96fa0a7d3457747222007490f039bc799